recipes/gplearn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gplearn" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/trevorstephens/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b4ef6139bb8407dd982310a2d227ef11af7a298d1ede82aeb2f0dd55ac3a7bf8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - scikit-learn >=0.22.1
+    - joblib >=0.13.0
+
+test:
+  imports:
+    - gplearn
+
+about:
+  home: https://github.com/trevorstephens/gplearn/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
`license_file: LICENSE`

Reply:
Sorry I missed this comment - yes I think I figured out what you wanted after checking some other feedstocks and PRs. I think this is resolved now - let me know if you're okay with it.